http-server-netty/src/test/groovy/io/micronaut/http/server/netty/RequestCancelSpec.groovy

```diff
@@ -0,0 +1,93 @@
+package io.micronaut.http.server.netty
+
+import io.micronaut.context.ApplicationContext
+import io.micronaut.context.annotation.Requires
+import io.micronaut.http.HttpResponse
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+import io.micronaut.http.annotation.RequestFilter
+import io.micronaut.http.annotation.ServerFilter
+import io.micronaut.runtime.server.EmbeddedServer
+import io.netty.buffer.Unpooled
+import io.netty.channel.embedded.EmbeddedChannel
+import io.netty.handler.codec.http.DefaultHttpRequest
+import io.netty.handler.codec.http.DefaultLastHttpContent
+import io.netty.handler.codec.http.FullHttpResponse
+import io.netty.handler.codec.http.HttpClientCodec
+import io.netty.handler.codec.http.HttpHeaderNames
+import io.netty.handler.codec.http.HttpMethod
+import io.netty.handler.codec.http.HttpObjectAggregator
+import io.netty.handler.codec.http.HttpResponseStatus
+import io.netty.handler.codec.http.HttpVersion
+import spock.lang.Specification
+
+import java.nio.charset.StandardCharsets
+
+class RequestCancelSpec extends Specification {
+    def 'filter returning early response'() {
```

Comment:
I assume this just checks the filter works, and not the change? 👍

Reply:
yes this is just to get some coverage of the deleted code. this test hits the branch that was deleted.